qutebrowser/browser/commands.py

```diff
@@ -1122,7 +1122,7 @@ def _run_userscript(self, cmd, *args, verbose=False):
         try:
             userscripts.run_async(tab, cmd, *args, win_id=self._win_id,
                                   env=env, verbose=verbose)
-        except userscripts.UnsupportedError as e:
+        except (userscripts.UnsupportedError, userscripts.NotFoundError) as e:
```

Comment:
@The-Compiler Is this style okay or would you prefer a common userscript exception base?